src/coreclr/inc/corinfo.h

```diff
@@ -2087,6 +2098,15 @@ class ICorStaticInfo
             CORINFO_METHOD_HANDLE       method
             ) = 0;
 
+    // return the entry point calling convention for any of the following
```

Comment:
Is this meant to deal with just unmanaged calling conventions? 

If yes, should this replace `getUnmanagedCallConv` method (ie `getUnmanagedCallConv` method should be deleted as part of this PR)?
If no, I think there is ambiguity for the PInvoke method case. PInvokes have both managed and unmanaged calling conventions.

Reply:
Yes, it is meant to deal with just unmanaged calling conventions. I'll delete the `getUnmanagedCallConv` method since is it now unused in this PR. I didn't know the policy for deleting unused methods in the jit interface.